greendao/src/main/java/com/epam/greendao/Model.java

```diff
@@ -0,0 +1,214 @@
+    private String company;
+    private String email;
+    private String about;
+    private String registered;
+    private Double latitude;
+    private Double longitude;
+
+    // KEEP FIELDS - put your custom fields here
+    // KEEP FIELDS END
+
+    public Model() {
+    }
+
+    public Model(String id) {
+        this.id = id;
+    }
+
+    public Model(String id, Integer index, Boolean isActive, String picture, String name, String company, String email, String about, String registered, Double latitude, Double longitude) {
+        this.id = id;
+        this.index = index;
+        this.isActive = isActive;
+        this.picture = picture;
+        this.name = name;
+        this.company = company;
+        this.email = email;
+        this.about = about;
+        this.registered = registered;
+        this.latitude = latitude;
+        this.longitude = longitude;
+    }
+
+    public Model(IEntity entity) {
+        this.id = entity.getId();
+        this.index = entity.getIndex();
+        this.isActive = entity.isActive();
+        this.picture = entity.getPicture();
+        this.name = entity.getEmployeeName();
+        this.company = entity.getEmployeeCompany();
+        this.email = entity.getEmployeeEmail();
+        this.about = entity.getEmployeeAbout();
+        this.registered = entity.getEmployeeRegisteredFormatted();
+        this.latitude = entity.getLatitude();
+        this.longitude = entity.getLongitude();
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public Integer getIndex() {
+        return index;
+    }
+
+    @Override
+    public Boolean isActive() {
+        return isActive;
+    }
+
+
+    public void setPicture(String picture) {
+        this.picture = picture;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getCompany() {
+        return company;
+    }
+
+    public void setCompany(String company) {
+        this.company = company;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public String getAbout() {
+        return about;
+    }
+
+    public void setAbout(String about) {
+        this.about = about;
+    }
+
+    public String getRegistered() {
+        return registered;
+    }
+
+    public void setRegistered(String registered) {
+        this.registered = registered;
+    }
+
+    public Double getLatitude() {
+        return latitude;
+    }
+
+    public void setLatitude(Double latitude) {
+        this.latitude = latitude;
+    }
+
+    public Double getLongitude() {
+        return longitude;
+    }
+
+    @Override
+    public String getTags() {
+        return null;
+    }
+
+    @Override
+    public void print() {
+        Log.d("GreenDAO", TextUtils.join(
```

Comment:
Please move this log to some Utils class which will be used across all modules!
The util method should take Tag and IEntity